cmd/restic/cmd_check.go

```diff
@@ -316,7 +333,7 @@ func runCheck(ctx context.Context, opts CheckOptions, gopts GlobalOptions, args
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		bar := newTerminalProgressMax(!gopts.Quiet, 0, "snapshots", term)
+		bar := newTerminalProgressMax(!gopts.Quiet && !gopts.JSON, 0, "snapshots", term)
```

Comment:
should use `bar := printer.NewCounter("snapshots")`. I forgot to convert the progress bars when switching `check` to `printer.Progress`.

Reply:
Done in 2b1686d6694cb0c5f676b9dadbb6c6c443c704f8.

Reply:
FYI: I cannot use this approach for "packs" as this uses `packCount` instead of `0`.